tests/app/utilities/test_schema.py

```diff
@@ -152,6 +156,32 @@ def test_load_schema_from_url_404():
     with pytest.raises(NotFound):
         load_schema_from_url(survey_url=TEST_SCHEMA_URL, language_code="en")
 
+    cache_info = load_schema_from_url.cache_info()
+    assert cache_info.currsize == 0
+    assert cache_info.hits == 0
```

Comment:
For completeness and visibility of the pattern is it worth asserting the misses too through each test?

Reply:
Done